iofog-agent-daemon/src/main/java/org/eclipse/iofog/microservice/MicroserviceState.java

```diff
@@ -18,7 +18,7 @@
  */
 public enum MicroserviceState {
 
-    QUEUED, PULLING, STARTING, RUNNING, STOPPING, DELETING, MARKED_FOR_DELETION, UPDATING, RESTARTING, STUCK_IN_RESTART, UNKNOWN, FAILED, STOPPED;
+    QUEUED, PULLING, STARTING, RUNNING, STOPPING, DELETING, MARKED_FOR_DELETION, UPDATING, RESTARTING, STUCK_IN_RESTART, UNKNOWN, FAILED, STOPPED, EXITED;
```

Comment:
What we were seeing with the AI msvcs was constant restarting. What needs to happen to trigger STUCK_IN_RESTART here? or RESTARTING? Rather than just EXITING..

Reply:
Updated